composer.json

```diff
@@ -23,17 +23,12 @@
         "php":  ">=5.3.0"
     },
     "require-dev":  {
-        "pimple/pimple": "~1.0",
```

Comment:
The deprecated Pimple 1 providers are now tested only in the job testing Silex too, through the Silex requirement. I considered it fine, as it is only for a long-deprecated class.